fiat_pair_former.py

```diff
@@ -0,0 +1,25 @@
+# In case we have access to the OHLCV data of two crypto/fiat pairs,
+# we can use this script to form the OHLCV data of the fiat pair we need
+
+## ASSUMPTION: The OHLCV data of the two pairs has the same length and the same timestamps,
+## which may not be the case. 
+
+import pandas as pd
+
+# Load the OHLCV data of the pair with the desired fiat currency
+csv_file_path_1 = 'ohlcv/ETHEUR_1.csv'
+# Load the OHLCV data of the pair with usd as base currency
+csv_file_path_2 = 'ohlcv/ETHUSD_1.csv'
+
+result_csv_file_path = 'EURUSD.csv'
+
+df1 = pd.read_csv(csv_file_path_1, header=None)
+df2 = pd.read_csv(csv_file_path_2, header=None)
+
+result_df = df1.iloc[:, [0]]
+
+divided_columns = df2.iloc[:, 1:] / df1.iloc[:, 1:]
```

Comment:
To be totally correct, the low and high prices would result from cross division instead of pointwise division (e.g., low EURUSD = low ETHUSD / high ETHEUR instead of low ETHUSD / low ETHEUR) but at the end the results would look even better than they already are – so we won't win a lot here.

Reply:
You are right! But keep in mind that this code was not used for analysis, since either the FIAT pairs were already on the data, or there were no FIAT/Crypto pairs to form the missing ones. I left the code just as a reminder in case we needed to form a FIAT/FIAT pair someday. 

Do you think it makes sense to keep it around or just delete it since it may create confusion?

Reply:
Ah okay, for me it was not clear how your code was used. It's probably good to either add some documentation to the code in this branch or document here in this non-mergable PR how those files have been used. I could figure that you first run the squid from scratch which then created a JSON file, which you then analyzed with one of the python scripts. Or is there this kind of documentation and I missed it?

Reply:
Yes the process is like that. The notion and the description of this PR should already explain the process. I will try to make it more clear.